index.js

```diff
@@ -63,6 +81,31 @@ function handleError( error, id, url ) {
 	return bot.sendMessage( id, error.message );
 }
 
+function sendUnfollowAcknowledgement( id, blog, count ) {
+	if ( count === 0) {
+		bot.sendMessage( id, `It seems you were not following ${blog}` );
+	} else {
+		bot.sendMessage( id, `No longer following ${blog}` );
+	}
+}
+
+function processCommand( id, command ) {
+	if ( command.method === 'follow' ) {
+		// we do not send a bot response yet:
+		// the response to xmpp sub command will trigger the response
+		return Promise.resolve()
+			.then( () => xmpp.subscribe( blogPath( command.blog ), id ) );
```

Comment:
If we decide to scale it to multiple xmpp accounts, we should ensure here we don't have other subscriptions for that blog in DB